client/app/components/BaseContainer/index.jsx

```diff
@@ -0,0 +1,109 @@
+// @flow
+import React from 'react';
+import { StoryContainer } from './StoryContainer';
+
+type A = { data: any, lastPage: any };
+
+export type Props = {
+  container: string,
+  data: any,
+  fetchUrl: string,
+  lastPage?: boolean,
```

Comment:
I think we can re-word this to `hasLastPage`, that implies more that the type is a boolean.